### ISSUE TITLE
Set default "methods supported" values for payment providers properly

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#3085]
### Expected behavior
If a payment provider does not support "Refund" that button should not be available.

### Actual Behavior
Refund button is shown for Authorize.net and others that don't support refunds and they show a message that says basically "We don't support refunds" when you try to process a refund.

### Steps to Reproduce the Behavior
1. Login as admin and configure shipping and Authorize.net as payment provider
1. Create an order and capture it
1. Try to process a refund
1. Observe you get a message saying we don't support refunds

The message was a step forward but now that we have these "Supported Methods" settings, they should be set correctly by default.

### Versions
```
Node: 7.10.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.15.2
Reaction: 1.5.0
Reaction branch: release-1.5.0
Docker: 17.09.0-ce
```

**Comment by codejockie** [comment]
@zenweasel is it just Authorize.net that doesn't support refunds? If there are others, could you let me know so I can handle them as well?

**Comment by zenweasel** [comment]
Authorize is the only one that doesn't support refunds at all.

**Comment by spencern** [comment]
Resolved by #3152
